src/visual.ts

```diff
@@ -499,6 +513,9 @@ module powerbi.extensibility.visual {
 
                 this.behavior.renderSelection(false);
             }
+
+            this.axisX.selectAll("text").style("font-size", this.data.settings.categoryAxis.fontSize);
```

Comment:
Move it to renderXAxisLabels() method

Reply:
It's not working if put to renderXAxisLabels